src/sonic-config-engine/minigraph.py

```diff
@@ -1234,8 +1235,13 @@ def parse_deviceinfo(meta, hwsku):
                     if hostname is not None:
                        key = "%s|%s" % (hostname.text, key)
                     sys_ports[key] = {"system_port_id": system_port_id, "switch_id": switch_id, "core_index": core_id, "core_port_index": core_port_id, "speed": speed, "num_voq": num_voq}
-
-    return port_speeds, port_descriptions, sys_ports
+            asic_sensor = device_info.find(str(QName(ns,"AsicSensors")))
```

Comment:
We don't have "AsicSensors" and the attributes in the production minigraph for SUP and Linecard.

Reply:
I think we should have a control parameter for eg **"controllable"** here in platform.json file to control this, https://github.com/sonic-net/sonic-buildimage/blob/2a2f3bdcddcf4b0261f3f2b21ae495e12fb1ffea/device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/platform.json#L36

If the controllable is true, for any of the ASIC* thermals, we can include the ASIC_SENSOR config in config_db. 
```
    "ASIC_SENSORS": {
        "ASIC_SENSORS_POLLER_INTERVAL": {
            "interval": "10"
        },
        "ASIC_SENSORS_POLLER_STATUS": {
            "admin_status": "enable"
        }
    },
```
The Poller interval could be controllable too if we can add it to thermals in platform.json. If this is agreeable, we can update/reach out to authors of https://github.com/sonic-net/SONiC/blob/master/doc/asic_thermal_monitoring_hld.md (REF : https://github.com/sonic-net/SONiC/pull/557)

@prgeor to comment, Any other platforms use ASIC_SENSOR config now?